tests/unit/test_rg_matrix.cpp

```diff
@@ -330,10 +360,20 @@ TEST_F(RGMatrixTest, RGMatrix_del) {
 	info = RG_Matrix_setElement_UINT64(A, x, i, j);
 	ASSERT_EQ(info, GrB_SUCCESS);
 
+	RG_Matrix_DM_nvals(&nvals, A);
+	ASSERT_EQ(nvals, 0);
+	RG_Matrix_DP_nvals(&nvals, A);
+	ASSERT_EQ(nvals, 1);
+
```

Comment:
I think this is too much, spreading calls to `RG_Matrix_DP_nvals` and `RG_Matrix_DM_nvals` all over the place,
please introduce a dedicated test which you perform all sort of operations and validate the results of `RG_Matrix_DP_nvals` and `RG_Matrix_DM_nvals`